<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_cookie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Click!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cursor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Gives 1 cookie per second.</string>
    </property>
    <property name="text">
     <string>Buy Cursor : 10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_grandma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Gives 5 cookies per second.</string>
    </property>
    <property name="text">
     <string>Buy Grandma : 25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_farm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Gives 10 cookies per second.</string>
    </property>
    <property name="text">
     <string>Buy Farm : 100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cookies : 0&lt;/p&gt;&lt;p&gt;Cursors : 0&lt;/p&gt;&lt;p&gt;Grandmas : 0&lt;/p&gt;&lt;p&gt;Farms : 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Gives 50 cookies per second.</string>
    </property>
    <property name="text">
     <string>Buy Mine: 1000</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
